pegasus_wrapper/key_value.py

```diff
@@ -72,6 +73,42 @@ def __call__(
         """
 
 
+@attrs(frozen=True)
```

Comment:
Do we want tests for this?

Reply:
@joecummings : yes, can you add one (on this branch?)